backend/app/models/invited_users.py

```diff
@@ -0,0 +1,34 @@
+from sqlalchemy import inspect
```

Comment:
you already mentioned this but leaving this note for me 😅 - invited_users table is no longer needed with the new users table